rotkehlchen/exchanges/binance.py

```diff
@@ -522,30 +526,108 @@ def _deserialize_asset_movement(self, raw_data: Dict[str, Any]) -> Optional[Asse
 
         return None
 
+    def _query_api_query_dict_within_time_delta(
+            self,
+            start_ts: Timestamp,
+            end_ts: Timestamp,
+            time_delta_ts: Timestamp,
+            method: str,
+    ) -> List[Dict[str, Any]]:
+        """Request via `api_query_dict()` from `start_ts` `end_ts` using a time
+        delta (offset) less than `time_delta_ts`.
+
+        Be aware of:
+          - If `start_ts` equals zero, the Binance launch timestamp is used
+          (from BINANCE_LAUNCH_TS). This value is not stored in the
+          `used_query_ranges` table, but 0.
+          - Timestamps are converted to milliseconds.
+          - This function does not currently support limits.
+        """
+        if method == 'depositHistory.html':
+            query_schema = 'depositList'
+        elif method == 'withdrawHistory.html':
+            query_schema = 'withdrawList'
+        else:
+            raise AssertionError(f'Unexpected binance method case: {method}.')
+
+        results: List[Dict[str, Any]] = []
+        # Create required time references in milliseconds
+        time_delta_ts_ms = time_delta_ts * 1000 - 1  # less than in ms
```

Comment:
nitpick: Please don't add the _ms in everything in this function. As you are noting here that you are using milliseconds it's fine. The code readers know that. It becomes hard to read the `ts_ms` prefix all around the function.